Pod/Classes/UI/CardTextField.swift

```diff
@@ -588,24 +596,24 @@ public class CardTextField: UITextField, NumberInputTextFieldDelegate {
         // Return false if any of this text field's subviews is already first responder. 
         // Otherwise let `numberInputTextField` become the first responder.
         if [numberInputTextField,monthTextField,yearTextField,cvcTextField]
-            .flatMap({return $0.isFirstResponder()})
-            .reduce(true, combine: {$0 && $1}) {
+            .flatMap({return $0.isFirstResponder})
+            .reduce(true, {$0 && $1}) {
             return false
         }
         return numberInputTextField.becomeFirstResponder()
     }
-    
-    public override func isFirstResponder() -> Bool {
+
+    public override func responds(to aSelector: Selector!) -> Bool {
```

Comment:
This should be `public override var isFirstResponder: Bool {`. Currently this causes a crash.